src/rustdoc-internals/htmldocck.md

```diff
@@ -0,0 +1,101 @@
+# `rustdoc` tests
+
+This page is specifically about the `rustdoc` test suite, for other test suites used for testing rustdoc, see [Rustdoc § Tests](../rustdoc.md#tests).
```

Comment:
While it's mentioned in `src/rustdoc.md` that `tests/rustdoc/` is specifically for testing the output of rustdoc's HTML backend; people arriving from `SUMMARY.md` won't necessarily know that given the (unfortunately) generic test suite name "`rustdoc`".

Therefore I'd mention that again here.